README.md

```diff
@@ -129,6 +129,18 @@ if err := gcslock.Unlock(m, 100*time.Millisecond); err != nil {
 }
 ```
 
+## Shell Script Usage
+
+You can also use this technique via the command line using the gsutil
+command, which enables serialization of shell scripts running anywhere.
+The `gcslock.sh` file defines two functions, which give the ability to
+globally lock shell script logic like this:
+```
+lock
```

Comment:
maybe add "source gcslock.sh" as the first line in the example?

you can also drop the triple backquote 

    ```

and prefix the example lines with 4 spaces, since you don't use syntax highlighting, or append "sh" to the backquotes to enable it:

    ```sh